interfaces/builtin/qualcomm_ipc_router.go

```diff
@@ -40,32 +45,214 @@ const qipcrtrBaseDeclarationSlots = `
   qualcomm-ipc-router:
     allow-installation:
       slot-snap-type:
+        - app
         - core
+    allow-connection:
+      plug-attributes:
+        qcipc: $SLOT(qcipc)
     deny-auto-connection: true
```

Comment:
I think we want something closer to content:

```
    allow-auto-connection:
      plug-publisher-id:
        - $SLOT_PUBLISHER_ID
      plug-attributes:
        qcipc: $SLOT(qcipc)
    deny-auto-connection:
      plug-attributes:
         qcipc: $MISSING
```

Reply:
Done